src/test-integration/java/org/opendcs/fixtures/configurations/opendcs/pg/OpenDCSPGConfiguration.java

```diff
@@ -74,30 +79,45 @@ public Map<String,String> getEnvironment()
      * Actually setup the database
      * @throws Exception
     */
-    private void installDb(SystemExit exit,EnvironmentVariables environment) throws Exception
+    private void installDb(SystemExit exit,EnvironmentVariables environment, UserPropertiesBuilder configBuilder) throws Exception
     {
-        if (started)
+        if (isRunning())
         {
             return;
         }
+        if(db == null)
+        {
+            db = new PostgreSQLContainer<>("postgres:15.3")
+                    .withUsername("dcs_owner")
+                    .withDatabaseName("dcs")
+                    .withPassword("dcs_owner");
+        }
+
+        db.start();
+        createPropertiesFile(configBuilder, this.propertiesFile);
         HashMap<String,String> placeHolders = new HashMap<>();
         placeHolders.put("NUM_TS_TABLES","1");
         placeHolders.put("NUM_TEXT_TABLES","1");
 
         Flyway flyway = Flyway.configure()
-                              .dataSource(dbUrl,null,null)
+                              .schemas("public")
+                              .dataSource(db.getJdbcUrl(),db.getUsername(),db.getPassword())
                               .placeholders(placeHolders)
                               .locations("db/opendcs-pg")
+                              .validateMigrationNaming(true)
                               .load();
 
         flyway.migrate();
-        Jdbi jdbi = Jdbi.create(dbUrl);
+        Jdbi jdbi = Jdbi.create(db.getJdbcUrl(),db.getUsername(),db.getPassword());
         jdbi.useHandle(h->{
             log.info("Creating application user.");
-            h.execute("create user dcs_proc with password 'dcs_proc'");
+            h.execute("DO $do$ begin create user dcs_proc with password 'dcs_proc'; exception when duplicate_object then raise notice 'user exists'; end; $do$");
             h.execute("GRANT \"OTSDB_ADMIN\" TO dcs_proc");
             h.execute("GRANT \"OTSDB_MGR\" TO dcs_proc");
             log.info("Setting authentication environment vars.");
+            environmentVars.put("DB_USERNAME","dcs_proc");
+            environmentVars.put("DB_PASSWORD","dcs_proc");
```

Comment:
I would normally expect reading password from environment instead of putting in environment, but I think this is for testing in a container only?